recipes/paho-mqtt-c/all/conanfile.py

```diff
@@ -14,11 +14,12 @@ class PahoMqttcConan(ConanFile):
     options = {"shared": [True, False],
                "fPIC": [True, False],
                "ssl": [True, False],
-               "asynchronous": [True, False]}
-    default_options = {"shared": False,
+               "samples": [True, False]}
+    # static builds didn't really work until 1.3.4
+    default_options = {"shared": True,
                        "fPIC": True,
-                       "ssl": False,
-                       "asynchronous": True}
```

Comment:
You can remove it, because it can break user which is using this option explicitly.

If it changed, you remove by config_options, but with a condition based on package version.

Reply:
Actually looking closer, this has not changed in the project.  But the original contribution of this package (not written by me) has a patch which builds either paho-mqtt3cs OR paho-mqtt3as.  Elsewhere I always build both libraries - they are essentially two difference APIs.  One is easier to use, the other more capable.  I'd prefer to include both libraries with every build as is done elsewhere.  What do you think about that?

Reply:
Asynchronous option removed.